ipywidgets_bokeh/widget.py

```diff
@@ -22,6 +22,7 @@ class IPyWidget(HTMLBox):
 
     def __init__(self, *, widget: Widget, **kwargs):
         super().__init__(**kwargs)
+        self._widget = widget
```

Comment:
Currently don't see a way around keeping this handle because I have to patch the custom kernel onto the `Widget.comm` when rendered in Panel.

Reply:
Actually since I wouldn't expect Panel users to have to construct the `IPyWidget` model themselves I can probably avoid this.